src/config.js

```diff
@@ -90,6 +90,7 @@ global.config = {
     numberOfNextroomers: 10,
     nextroomerInterval: 500,
     maxRooms: 20,
+    cpuPerRoom: 12, // Necessary CPU per room, prevent claiming new rooms
```

Comment:
i just did a respawn on monday, on my first room i had cpu peeks up to 55 cpu, with 2 i have an average so this could be too low? 
// so is cpuPerRoom: 20 too high? 
// or so cpuPerRoom: _.floor(Game.cpu.limit - 20) / 5,

@TooAngel just btw should not be maxRooms: Game.gcl.level, ?

Reply:
`maxRooms` was my first approach to reduce the number of rooms to fit somehow to my `cpu.limit` level and to introduce some mechanism for skipping.
`cpuPerRoom` is my next approach, next step, measure the CPU usage and only claim a room when enough CPU is available.

I'm thinking about increasing to 13. You start with 30 CPU and get per each GCL one more.
With 20 you can have 3 rooms and on GCL 4 you have to wait a full gcl, IMHO this is too early, especially for the bot or new players.
I think ~13 CPU is a limit we could eye for in regards to CPU optimizations. What do you think?